src/shared/tpm2-util.c

```diff
@@ -596,8 +598,65 @@ static int tpm2_get_best_pcr_bank(
         return 0;
 }
 
+static int tpm2_make_hmac_session(
+                ESYS_CONTEXT *c,
+                ESYS_TR tpmKey,
+                ESYS_TR *ret_session) {
+
+        static const TPMT_SYM_DEF symmetric = {
+                .algorithm = TPM2_ALG_AES,
+                .keyBits = {
+                        .aes = 128
+                },
+                .mode = {
+                        .aes = TPM2_ALG_CFB,
+                }
+        };
+        ESYS_TR session = ESYS_TR_NONE;
+        TSS2_RC rc;
+
+        assert(c);
+
+        log_debug("Starting HMAC session.");
```

Comment:
i think a comment here what this does and what it is for would be more than welcome.

Reply:
(i.e. copy tthe stuff from the commit msg here)